File: TheMovie/app/src/main/java/com/nyinyi/nw/themovie/network/RetrofitDataAgent.java
package com.nyinyi.nw.themovie.network;

/**
 * Created by User on 9/14/2017.
 */

public interface RetrofitDataAgent {

    void loadNowPlayingMovies();

    void loadUpcomingMovies();

    void loadPopularMovies();

    void loadMovieDetail(String movie_id);

}

File: TheMovie/app/src/main/java/com/nyinyi/nw/themovie/network/RetrofitDataAgent.kt
package com.nyinyi.nw.themovie.network

/**
 * Created by User on 9/14/2017.
 */

interface RetrofitDataAgent {

    fun loadNowPlayingMovies()

    fun loadUpcomingMovies()

    fun loadPopularMovies()

    fun loadMovieDetail(movie_id: String)

}
